pallets/gear/src/lib.rs

```diff
@@ -337,6 +343,85 @@ pub mod pallet {
     where
         T::AccountId: Origin,
     {
+        /// Submit program for benchmarks which does not check nor instrument the code.
+        #[cfg(feature = "runtime-benchmarks")]
+        pub fn submit_program_raw(
```

Comment:
Could some common extracted to the new function so `submit_program` & `submit_program_raw` could call it?